package.json

```diff
@@ -12,9 +12,7 @@
     "postrelease": "npm publish && git push --follow-tags",
     "pretest": "standard",
     "release": "standard-version -s",
-    "test": "tap -J --100 --coverage test/*.js",
-    "update-coc": "weallbehave -o . && git add CODE_OF_CONDUCT.md && git commit -m 'docs(coc): updated CODE_OF_CONDUCT.md'",
-    "update-contrib": "weallcontribute -o . && git add CONTRIBUTING.md && git commit -m 'docs(contributing): updated CONTRIBUTING.md'"
+    "test": "tap -J --100 --coverage test/*.js"
```

Comment:
Once tap is updated to latest, this can just be `tap test/*.js`, or move any non-test stuff in test/ to `test/fixtures` and make this script just `"test": "tap"`